file-browser/package.json

```diff
@@ -22,17 +22,19 @@
   },
   "private": true,
   "dependencies": {
+    "electron-is-dev": "^0.2.0",
+    "eywa": "git+https://github.com/escherpad/eywa.git",
     "layout-components": "^2.7.1",
     "luna": "^1.5.0",
-    "luna-saga": "^3.1.0",
+    "luna-saga": "^3.0.1",
```

Comment:
this seems wrong. Why don't we stay on 3.1.0? would you please double check this? Thanks!

- [ ] make sure that @3.1.0 works.
- [ ] remote this edit.